### PR TITLE
Snackbar

### PR DESCRIPTION
**Pull request by saifudinbadani**
Snackbar component added.

snackbar

### CODE REVIEW

File: components/snackbar/snackbar.css [hunk 1]
```diff
@@ -0,0 +1,15 @@
+@import "../../shared.css";
+
+.snackbar{
+    background-color: var(--orange-500);
+    color: var(--secondary);
+    display: inline-block;
+    border-radius: 0.5rem;
```
Comment (tushargoyal812) [inline components/snackbar/snackbar.css:7]: suggestion:use px for border radius

**Review by tushargoyal812 — COMMENTED** [review]
good folder structure looks good to merge

**Review by chanchal-panpaliya — COMMENTED** [review]
class name is consistent and readable and overall coding structure is also good .

File: components/snackbar/snackbar.css (hunk 1 above)
Comment (chanchal-panpaliya) [reply components/snackbar/snackbar.css:7]: I'm not agree with this . read this : - [https://stackoverflow.com/questions/23991054/does-rem-make-sense-on-border-radius](url)

**Pull request merged**